acceptancetests/assess_caas_deploy_charms.py

```diff
@@ -181,19 +196,24 @@ def assess_caas_charm_deployment(client):
         juju_ver=client.version
     )
 
-    caas_client = deploy_caas_stack(bundle_path=bundle, client=client)
+    caas_client = deploy_caas_stack(path=bundle, client=client, timeout=4000)
     external_hostname = caas_client.get_external_hostname()
 
     if not caas_client.is_cluster_healthy:
         raise JujuAssertionError('k8s cluster is not healthy because kubectl is not accessible')
 
     # tmp fix kubernetes core ingress issue
-    caas_client.kubectl(
-        'patch', 'daemonset.apps/nginx-ingress-kubernetes-worker-controller', '--patch',
+    ingress_controller_daemonset_name = 'daemonset.apps/nginx-ingress-kubernetes-worker-controller'
+    o = caas_client.kubectl(
```

Comment:
well, I would prefer a very short name rather than a very detailed long name if the var is just used in the next line rather somewhere else and its name does not have to be meaningful to make code cleaner(less interruption when u read the code).